pkg/log/handler_iowriter.go

```diff
@@ -13,11 +13,17 @@ func init() {
 }
 
 type HandlerIoWriterSettings struct {
-	Level           string   `cfg:"level" default:"info"`
-	Channels        []string `cfg:"channels"`
-	Formatter       string   `cfg:"formatter" default:"console"`
-	TimestampFormat string   `cfg:"timestamp_format" default:"15:04:05.000"`
-	Writer          string   `cfg:"writer" default:"stdout"`
+	Level           string    `cfg:"level" default:"info"`
+	Channels        []Channel `cfg:"channels"`
+	Formatter       string    `cfg:"formatter" default:"console"`
+	TimestampFormat string    `cfg:"timestamp_format" default:"15:04:05.000"`
+	Writer          string    `cfg:"writer" default:"stdout"`
+}
+
+type Channel struct {
+	Name     string `cfg:"name"`
+	Level    string `cfg:"level" default:"info"`
+	Disabled bool   `cfg:"disabled" default:"false"`
```

Comment:
Hmm, this completely changes the way the channel filter works. Previously one would simply specify the channels to filter for, so not specifying any channel would filter it out. Now we always log all channels unless a channel is explicitly disabled, so there is no way to filter out all channels unless one (as there are no regex or glob patterns one could use).

And if we go with a flag, I would prefer an enabled flag instead I think